drivers/include/periph/flashpage.h

```diff
@@ -107,6 +123,29 @@ static inline int flashpage_page(void *addr)
 void flashpage_write(int page, void *data);
 
 /**
+ * @brief   Write the given address with the given data and length
+ *
+ * @warning Make sure the targeted memory area is erased before calling
+ *          this function
+ *
+ * The given address must be aligned to @p FLASHPAGE_RAW_BLOCKSIZE.
```

Comment:
rewrite: "both target address and data address must be aligned to FLASHPAGE_RAW_ALIGN".

add "len must be a multiple of FLASHPAGE_RAW_BLOCKSIZE"

Reply:
Done.